mlir/test/Integration/GPU/CUDA/sm90/tma_load_64x8_8x128_noswizzle.mlir

```diff
@@ -6,15 +6,6 @@
 // RUN:   --entry-point-result=void \
 // RUN:  | FileCheck %s
 
-// Basic PTX check to make sure we are generating the right instructions.
-
-// CHECK-PTX: mbarrier.init.shared.b64
```

Comment:
This was changed here `--check-prefixes=CHECK-PTX` https://github.com/llvm/llvm-project/pull/68184/files#diff-468d25896ab3abaf617d05a64f7d107372665c6a6a9af470207dddd0f1f66a6d; should be `CHECK-PTX`  prefix added back or kill all lines 1-8?

Reply:
There are merits of checking the PTX assembly. But this is an integration test. It's not the best place the test PTX. I've added that check earlier, but we now test the generated PTX in the conversion tests.